src/backend/core/services/quota_redis.py

```diff
@@ -0,0 +1,278 @@
+"""
+Redis-based recipient quota service.
```

Comment:
the fact that it's based on redis should be an implementation detail and not appear in the filename or public interface